### PR TITLE
LP-2285 Fixing marketplace linting issues

### PR DESCRIPTION
**Pull request by zahid-arbisoft**
### Description

[LP-2285](https://philanthropyu.atlassian.net/browse/LP-2285) Fixing marketplace linting issues

### Notes
All Pylint issue fixed score is 10/10
Isort ran on marketplaces files

### CODE REVIEW

File: openedx/features/marketplace/tests/factories.py [hunk 1]
```diff
@@ -8,7 +11,7 @@
 class ChallengeFactory(OrganizationBaseFactory, LocationFactory):
     """Factory for MarketplaceRequest model. It contains fake data or sub-factories for all mandatory fields"""
 
-    class Meta:
+    class Meta(object):
```
Comment (asadali145) [inline openedx/features/marketplace/tests/factories.py:14]: Any specific reason for this change?

**Review by asadali145 — APPROVED** [review]
Just a small question, Apart from that LGTM

File: openedx/features/marketplace/tests/factories.py (hunk 1 above)
Comment (zahid-arbisoft) [reply openedx/features/marketplace/tests/factories.py:14]: Yes, it was highlighted in pylint report (you can see the complete [report on JIRA ](https://philanthropyu.atlassian.net/browse/LP-2285?focusedCommentId=18884)story)

`openedx/features/marketplace/tests/factories.py:11: [C1001(old-style-class), ChallengeFactory.Meta] Old-style class defined.`

**Review by ahmed-faraz46 — COMMENTED** [review]
LGTM 👍

**Review by ahmed-faraz46 — APPROVED** [review]

**Comment by zahid-arbisoft** [comment]
Merging PR. We can ignore the quality check for now, as it is failing due to error/bug

**Pull request merged**